electricpy/__init__.py

```diff
@@ -5077,4 +5077,50 @@ def pi_attenuator(Adb, Z0):
     R2 = (Z0/2)*(_np.power(10, x) - (1/(_np.power(10, x))))
 
     return R1,R2
+
+
+def lm317(r1, r2, v_out):
+    r"""
+    LM317.
+
+    The LM317 is a linear voltage regulator that can be adjusted to supply a specific output voltage.
+    The LM317 has three pins, adjust, output and input. The LM317 is often connected as in the image below.
```

Comment:
Should there be an image here? Something like:

```
.. image:: https://www.basictables.com/media/lm317-circuit.png
```

Reply:
Fixing them right away